code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -124,3 +124,11 @@
 		icon_state = "biofoam1"
 	else
 		icon_state = "biofoam0"
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/saltwater
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/saltwater/New()
+	..()
+	reagents.remove_reagent(DOCTORSDELIGHT, 30)
```

Comment:
disgusting

Reply:
that is exactly how the biofoam injectors work and how I was told to do so by pomf

Reply:
it's true

Reply:
In a righteous, good world someone would rewrite hypospray and autoinjector code, but this probably would involve mapping and we already have the mapping pipeline clogged with American Lights

Reply:
it's an easier change, gimme 10 min